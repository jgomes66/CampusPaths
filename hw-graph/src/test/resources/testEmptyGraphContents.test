# Test Empty Graph Contents
# Tests an empty graph has zero contents with listNodes method

# Create a graph
CreateGraph graph1

# List all nodes in graph
ListNodes graph1