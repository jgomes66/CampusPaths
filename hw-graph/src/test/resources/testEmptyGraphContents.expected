# Test Empty Graph Contents
# Tests an empty graph has zero contents with listNodes method

# Create a graph
created graph graph1

# List all nodes in graph
graph1 contains: